src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({
+  a = 0,
+  b = 0,
+}) {
```

Comment:
for the future, better use destructuring INSIDE function code, props may get big and bulky and hard to manage in function params